### ISSUE TITLE
Make GravityBee compatible with Anaconda

### ISSUE THREAD

**Issue by YakDriver** [plus3it/gravitybee#82]
**Is your feature request related to a problem? Please describe.**
GravityBee fails to build applications when used with Anaconda virtual environments.

**Describe the solution you'd like**
Make GravityBee work seamlessly in Anaconda environments.

**Describe alternatives you've considered**
Currently seeking testing of simple workarounds setting environment variables.

**Additional context**
See [this comment](https://github.com/plus3it/gravitybee/issues/74#issuecomment-536398060) in issue #74

**Comment by nikolarobottesla** [comment]
Hi, I'm using a conda environment and I tried setting the environment variables. The error I get now is "FileNotFoundError: [Errno 2] No such file or directory: 'entry_points.txt'" which seems to be coming from pyppynn.

`removing .\.pyppyn\temp
Traceback (most recent call last):
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\a1rc7zz\Miniforge3\envs\needlecast\Scripts\gravitybee.exe\__main__.py", line 7, in <module>
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\site-packages\click\core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\site-packages\click\core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\site-packages\click\core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\site-packages\gravitybee\cli.py", line 192, in main
    args = gravitybee.Arguments(**kwargs)
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\site-packages\gravitybee\__init__.py", line 204, in __ini
t__
    self.info["console_script"] = self.pyppy.get_config_attr(
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\site-packages\pyppyn\__init__.py", line 475, in get_confi
g_attr
    self.load_config()
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\site-packages\pyppyn\__init__.py", line 371, in load_conf
ig
    self.read_config()
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\site-packages\pyppyn\__init__.py", line 293, in read_conf
ig
    self._wheel_console_scripts()
  File "c:\users\a1rc7zz\miniforge3\envs\needlecast\lib\site-packages\pyppyn\__init__.py", line 224, in _wheel_co
nsole_scripts
    with open("entry_points.txt", "r") as ep_file:
FileNotFoundError: [Errno 2] No such file or directory: 'entry_points.txt'
`

**Comment by lorengordon** [comment]
@nikolarobottesla Did you follow the linked threads in the OP issue to setup your virtual env?

**Comment by nikolarobottesla** [comment]
I think so I did the following:
> export VIRTUAL_ENV="$HOME/Miniforge3/envs/needlecast"
> export PATH="$VIRTUAL_ENV/bin:$PATH"
> unset PYTHON_HOME

**Comment by lorengordon** [comment]
You seem to have some conflicting setup... some things look like Windows (e.g. paths using `\`), and some things look like Linux (`export`, `$HOME`, forward slash paths `/`). We are definitely not working actively to support Conda envs. But we are absolutely open to PRs. If you can get it working, please open a PR. If you'd like examples of how we setup to run the utility, see the CI configs at [`azure-pipelines.yml`](https://github.com/plus3it/gravitybee/blob/master/azure-pipelines.yml) or [`.travis.yml`](https://github.com/plus3it/gravitybee/blob/master/.travis.yml).

**Comment by nikolarobottesla** [comment]
The problem I posted about earlier was due to a lack of an entry_point.  Anyways I worked through that and kind of got it working with conda environments.

I also have a non-standard package directory and I'm also using the 'package_dir' variable in setup() in setup.py to indcate the package location. When I manually run pyinstaller in the same directory as my entry_point python file it builds the exe correctly. When I run gravitybee or pyinstaller from the top level directory it builds an executible but the exe is missing the imports from the package's folder.

The gravitybee log shows the missing python scripts being copied to the different directories but pyinstaller still doesn't include them for some reason.
